cmake/Modules/G4DeveloperAPI_OLD.cmake

```diff
@@ -238,7 +238,6 @@ macro(geant4_library_target)
     # - Common shared lib commands
     add_library(${G4LIBTARGET_NAME} SHARED ${G4LIBTARGET_SOURCES})
     target_compile_definitions(${G4LIBTARGET_NAME} PRIVATE GEANT4_DEVELOPER_$<CONFIG>)
-    target_compile_features(${G4LIBTARGET_NAME} PUBLIC ${GEANT4_TARGET_COMPILE_FEATURES})
```

Comment:
Should be retained, compile features are the way we (should) handle definitions of how to compile things.